src/platforms/react-native/upload-debug.mdx

```diff
@@ -16,6 +16,10 @@ Errors raised from the native layer in Android apps require certain debug inform
 
 Upload the [Android ProGuard/R8 mapping files and native symbols](/platforms/android/proguard/) by the recommended method of using our Gradle integration.
 
+## Uploading With Xcode
+
+Build phase [`Upload Debug Symbols to Sentry`](/platforms/react-native/manual-setup/manual-setup/#upload-debug-symbols-to-sentry) uses `sentry-cli` to upload native iOS debug symbols. To enable [Source Context](/platforms/apple/guides/ios/data-management/debug-files/source-context/) you can use `SENTRY_INCLUDE_NATIVE_SOURCES=true` environmental variable set before build.
```

Comment:
```suggestion
Build phase [`Upload Debug Symbols to Sentry`](/platforms/react-native/manual-setup/manual-setup/#upload-debug-symbols-to-sentry) uses `sentry-cli` to upload native iOS debug symbols. To enable [Source Context](/platforms/apple/guides/ios/data-management/debug-files/source-context/), set `SENTRY_INCLUDE_NATIVE_SOURCES=true` environmental before build.
```

Reply:
`l`: maybe also link to https://docs.sentry.io/platforms/apple/dsym/#sentry-cli

Reply:
Maybe we can link it right away in the manual setup docs.
https://github.com/getsentry/sentry-docs/pull/7151